helix-core/src/test/java/org/apache/helix/common/ZkTestBase.java

```diff
@@ -719,6 +737,17 @@ public void cleanupLiveInstanceOwners() {
       clientMap.clear();
     }
     _liveInstanceOwners.clear();
+
+    boolean status = false;
+    try {
+      status = ThreadLeakageChecker.afterClassCheck(testClassName);
```

Comment:
Just to make sure, having this check here should cover all tests?

Reply:
Only those test derived from ZkTestBase. That is majority for now. We can open another ticket to track using then for all.